website/docs/sdk-reference/js-ssr.md

```diff
@@ -286,6 +286,28 @@ let configCatClient = configcat.createClient("#YOUR-SDK-KEY#");
 const keys = await configCatClient.getAllKeysAsync();
 console.log(keys);
 ```
+
+## `getAllValues()`
+
+You can evaluate all keys with `getAllValues()` method.
```

Comment:
maybe with *the*

Reply:
`Evaluates and returns the values of all feature flags and settings. Passing a [User Object](link to user object page in docs) is optional.`